docs/sphinx/developers/building-bioformats.txt

```diff
@@ -125,13 +105,6 @@ Alternatively, run the Eclipse Maven target with :command:`mvn
 eclipse:eclipse` to create the Eclipse project files, then use
 :menuselection:`File --> Import --> Existing Projects into Workspace`.
```

Comment:
This should be updated from `File --> Import --> Existing Projects into Workspace`. to `File --> Import --> Existing Maven Projects`.